lib/pagseguro/items.rb

```diff
@@ -13,15 +13,19 @@ def initialize
     def <<(item)
       item = ensure_type(Item, item)
 
-      if include?(item)
-        item.quantity += 1
```

Comment:
People take care with this kind of behavior :boom::boom::boom:.

This is absolutely wrong!